spec/models/pseud_spec.rb

```diff
@@ -16,4 +16,42 @@
 
   # TODO: add more tests
 
-end
\ No newline at end of file
+  describe "save" do
+
+		before do
+			@user = User.new
+      @user.login = "mynamepseud"
+      @user.age_over_13 = "1"
+      @user.terms_of_service = "1"
+      @user.email = "foo1@archiveofourown.org"
+      @user.password = "password"
```

Comment:
It shows up in my email with before do and @user = User.new indented with tabs, if that helps!

> On Jan 17, 2016, at 4:41 PM, Cesy <notifications@github.com> wrote:
> 
> In spec/models/pseud_spec.rb <https://github.com/otwcode/otwarchive/pull/2264#discussion_r49952964>:
> 
> > \ No newline at end of file
> > +  describe "save" do
> > +
> > +		before do
> > +			@user = User.new
> > +      @user.login = "mynamepseud"
> > +      @user.age_over_13 = "1"
> > +      @user.terms_of_service = "1"
> > +      @user.email = "foo1@archiveofourown.org"
> > +      @user.password = "password"
> Any ideas what it's seeing that I'm not? This looks consistent to me.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/otwcode/otwarchive/pull/2264/files#r49952964>.
>